docs/automl-capabilities/index.md

```diff
@@ -20,6 +20,63 @@ val features = Seq(emailFeature, phone, age, subject, zipcode).transmogrify()
 
 For advanced users, you can also completely [customize automatic feature engineering](../developer-guide#transmogrification).
 
+## NLP - NER Detection
+
+This is the stage which can be used in feature engineering to detect NERs in a sentence.
+
+It can be done in the following steps:
+
+**Token generation**
+
+We will use NameEntityRecognizer tokenize the input text. It is  NameEntityType text recognizer class which encapsulates 
```

Comment:
Perhaps add a note that users need add an extra dependency to their projects:
```
// TransmogrifAI pretrained models, e.g. OpenNLP POS/NER models etc. (optional)
compile 'com.salesforce.transmogrifai:transmogrifai-models_2.11:0.6.1'
```

Reply:
added